creating_S3_backend/main.tf

```diff
@@ -0,0 +1,26 @@
+provider "aws" {
```

Comment:
This is nothing more than the sample module copied - needs to use the terraform [Terraform Local File](https://registry.terraform.io/providers/hashicorp/local/latest/docs/resources/file) with [Terraform Template](https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file) to achieve the output file which has the variables inside.